app/src/main/java/org/mozilla/fenix/trackingprotection/TrackingProtectionPanelView.kt

```diff
@@ -151,7 +151,7 @@ class TrackingProtectionPanelView(
 
     private fun getLastUsedCategoryView(categoryTitle: String) = when (categoryTitle) {
         CROSS_SITE_TRACKING_COOKIES.name -> {
-            cross_site_tracking
+            if (cross_site_tracking.isGone) cross_site_tracking_loaded else cross_site_tracking
```

Comment:
This is a bit confusing to me 🤔 could you possibly add a clarifying comment about this (temporary?) behavior? Is this just while the feature flag is on that we are intertwining these two categories?

Reply:
This method is checking whether the permission was allowed-- titles ending in `.._loaded`-- or blocked when they were last used. Since we added cross site tracking as an option, it can either be in the allowed or blocked state. What was there before meant it wasn't ever showing as `_loaded`. 

I can add a comment to this method so it's more clear 👍